### DISCUSSION TITLE
Is There any way that i can update scope in existing app without reinstall app

### DISCUSSION THREAD

**Discussion by ghanshyam1994** [Kyon147/laravel-shopify#275] [Q&A]
Like api 

curl -X POST -H "Content-Type: application/json" -H "X-Shopify-Access-Token: XXXXXXXXXXXXXXX" -d '{"scope":"read_customers, write_customers, read_orders, read_discounts, write_discounts, write_orders, read_analytics"}' "https://store.myshopify.com/admin/api/2024-01/access_scopes/update.json"

I want to insert more access_scopes here

**Comment by Kyon147** [comment]
You can edit the scopes in your ENV or the `shopify-config` file and then it will ask them to accept new permissions the next time they log in.

Just make sure you are on the latest version and add the new middleware https://github.com/Kyon147/laravel-shopify/wiki/Upgrading#v19xx--v1920
